packages/uniforms/__tests__/connectField.tsx

```diff
@@ -119,100 +132,111 @@ describe('connectField', () => {
     it('treats value as initial value', async () => {
       const Field = connectField(Test);
 
-      mount(<Field name="field" value="initialValueExample" />, reactContext);
+      render(
+        <Field name="field" value="initialValueExample" />,
+        schema,
+        reactContext,
+      );
+
       await new Promise(resolve => setTimeout(resolve, 10));
 
       expect(onChange).toBeCalledWith('field', 'initialValueExample');
     });
   });
 
   describe('when rendered with label', () => {
-    const labelA = <span style={{ color: 'red' }}>Error</span>;
-    const labelB = <span style={{ color: 'green' }}>OK</span>;
-
     it.each([
       ['Props', '', 'Props'],
       ['Props', 'Schema', 'Props'],
-      ['Props', labelB, 'Props'],
       ['Props', undefined, 'Props'],
       ['', undefined, ''],
       ['', 'Schema', ''],
-      ['', labelB, ''],
       ['', undefined, ''],
-      [labelA, '', labelA],
-      [labelA, 'Schema', labelA],
-      [labelA, labelB, labelA],
-      [labelA, undefined, labelA],
       [undefined, '', ''],
       [undefined, 'Schema', 'Schema'],
-      [undefined, labelB, labelB],
       [undefined, undefined, ''],
```

Comment:
Why did you remove those cases?

Reply:
I created `Test` component that use `label` as `input` props and it couldn't be rendered.

I changed it now to render `label` above `input`. Rendering `label` is based on rendering `label` in `TextField` in the `uniforms-unstyled`. I needed to change `works with nested labels` test after this change.